tools/model_generator/model_gen.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"html/template"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+type columnIndex int
+
+const (
+	Name columnIndex = iota
+	InternalName
+	FieldType
+	Description
+	GroupName
+	FormField
+	Options
+	ReadOnlyValue
+	ReadOnlyDefinition
+	Calculated
+	ExternalOptions
+	Deleted
+	HubspotDefined
+	CreatedUser
+	Usages
+)
+
+func main() {
+	log.Printf("Running with arges: %s\n", os.Args[1:]) // Without command name.
+
+	if len(os.Args) != 3 {
+		log.Fatal("Missing required parameters: <objectName> <csvFilePath>")
```

Comment:
I think cli tool is better not to use log, but use fmt.Print* because log has date&time prefix.

Reply:
Example: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/cmd.go#L140